ES/ESmy018.xml

```diff
@@ -32,86 +32,185 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                  <summary>
+                    I) Taʾammǝra Māryām (fols. 3ra-43vb); I-1) Introductory exhortation
                      (fols. 3ra-4va)&#xD; I-2) Taʾammǝra Māryām: 30 miracles (fols. 4va-43vb)&#xD; II)
                      Discourse by John Chrysostom on the life and decollation of John the Baptist
-                     (fol. 44ra-60va) &#xD; III) Homily on the finding of the head of John the
-                     Baptist (fols. 60vb-65ra)&#xD; IV) Homily by John Chrisostom on the glory of
-                     John the Baptist (65ra-74ra)&#xD; &#xD; Additiones 1-2; colophons
-                     1-2.</summary>
-                  <msItem xml:id="ms_i1">
+                     (fol. 44ra-60va) &#xD; I</summary>
 
-                     <incipit>
-                        <foreign xml:lang="en"/>
-                     </incipit>
-                     <explicit>
-                        <foreign xml:lang="en"/>
-                     </explicit>
-                     <note/>
+                    <msItem xml:id="ms_i1">
+                     <locus from="3ra" to="43vb"/>
+                     <title type="complete" ref="LIT2384Taamme"/>
+
+                     <msItem xml:id="ms_i1.1">
+                      <locus from="3ra" to="4va"/>
+                      <title type="complete" ref="LIT3584Intro"/>
+                      <incipit xml:lang="gez">
+                         <locus target="#1r"/><hi rend="rubric">ስምዑ፡ እነግረክሙ፡ ኦአኃዊነ፡</hi> ውሉደ፡
+                         ቤተ፡ <sic>ክርስቲ</sic> እግዚአብሔር፡ ይባርክሙ፡ ወያብርህ፡ አዕይንተ፡ ልብክሙ፡ ለሰሚዐ፡ ተአምሪሃ፡
+                         ለእግዝእትነ፡ <hi rend="rubric">ማርያም፡</hi> ወላዲተ፡ አምላክ፡ ዘኢይክሉ፡ ሰማያውያን፡ ወምራውያን፡ ፈጽሞ፡ ውዳሴሃ፡ ኢሰማያውያን፡ ወኢምድራውያን፡
+                      ጸሎታ፡ ወበረከታ፡ ይሀሎ፡ ምስለ፡ ገብሩ፡ ተክለ፡ እዎስጣቴዎስ፨ ዝውእቱ፡ ቅድመ፡ ተአምር፡ ይትነበብ፡
+                      </incipit>
+                      </msItem>
+
+                      <msItem xml:id="ms_i1.2">
+                       <locus from="3ra" to="4va"/>
+                       <title type="complete" ref="LIT3584Intro"/>
+                     <note>Miracles</note>
                   </msItem>
-                  <!--The data in the items below refers to other items. please remove this items and move the information to the appropriate place.-->
-                  <msItem xml:id="ms_i2">
+                  <msItem xml:id="ms_i1.">
+                   <locus from="3ra" to="4va"/>
+                   <title type="complete" ref="LIT3584Intro"/>
+                 <note>Miracles</note>
+              </msItem>
+
+                  <msItem xml:id="ms_i1.30">
                      <note>The last miracle (<locus target="#43ra">43ra-</locus>va) is added by a
                         different, crude hand. The incipit of Text IV is not rubricated.</note>
                   </msItem>
+
                   <colophon xml:id="coloph1">
                      <locus target="#34va">34va</locus>, l.8: ዛቲ፡ መጽሐፍ፡ ዘአጽሐፋ፡ ገብረ፡ ክርስቶስ፡ ወልደ፡ ቀሲስ፡
```

Comment:
it should be
`<locus from="34va" to "34vb">34va, l. 8-34vb</locus>`
(l. 8 inside of locus, no colon, the range together for the colophon)